Headlines/src/Managers/UserSettingsManager.swift

```diff
@@ -9,53 +9,53 @@
 import UIKit
 
 class UserSettingsManager: NSObject {
-    
+
     var whitelistedSources: [String]? {
         set {
             UserDefaults.standard.setValue(newValue, forKey: "whitelisted_sources")
             UserDefaults.standard.synchronize()
         }
-        
+
         get {
             return UserDefaults.standard.stringArray(forKey: "whitelisted_sources")
         }
     }
-    
+
     var shouldOpenNewsInsideApp: Bool {
         guard let value = UserDefaults.standard.value(forKey: "open_news_inside_app") as? Bool else {
             return true
         }
         return value
     }
-    
+
     var firstOpenDate: Date? {
         set {
             UserDefaults.standard.setValue(newValue, forKey: "first_open_date")
             UserDefaults.standard.synchronize()
         }
-        
+
         get {
             return UserDefaults.standard.value(forKey: "first_open_date") as! Date?
         }
     }
-    
+
     var canceledReviewDate: Date? {
         set {
             UserDefaults.standard.setValue(newValue, forKey: "canceled_review_date")
             UserDefaults.standard.synchronize()
         }
-        
+
         get {
             return UserDefaults.standard.value(forKey: "canceled_review_date") as! Date?
         }
     }
-    
+
     var reviewDate: Date? {
         set {
             UserDefaults.standard.setValue(newValue, forKey: "review_date")
             UserDefaults.standard.synchronize()
         }
-        
+
         get {
             return UserDefaults.standard.value(forKey: "review_date") as! Date?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f5a5ab936ee0d164c160af7374b637afef6eb821/Headlines/src/Managers/UserSettingsManager.swift#L60"></span>Force casts should be avoided.<br />
<code>force_cast</code> <code>UserSettingsManager.swift:60</code></td>
    </tr>
  </tbody>
</table>